app/src/main/java/com/sopt/smeem/presentation/mypage/components/MyBadgesContent.kt

```diff
@@ -0,0 +1,159 @@
+package com.sopt.smeem.presentation.mypage.components
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+import com.sopt.smeem.R
+import com.sopt.smeem.data.datasource.BadgeList
+import com.sopt.smeem.domain.model.mypage.MyBadges
+import com.sopt.smeem.presentation.compose.theme.SmeemTheme
+import com.sopt.smeem.presentation.compose.theme.Typography
+import com.sopt.smeem.presentation.compose.theme.gray100
+import com.sopt.smeem.util.previewPlaceholder
+
+@Composable
+fun MyBadgesContent(
+    modifier: Modifier = Modifier,
+    badges: List<MyBadges>,
+    onClickCard: (MyBadges) -> Unit
+) {
+    SmeemContents(
+        title = stringResource(R.string.my_badges)
+    ) {
+        LazyVerticalGrid(
+            modifier = modifier.fillMaxWidth(),
+            columns = GridCells.Fixed(3),
+            verticalArrangement = Arrangement.spacedBy(7.dp),
+            horizontalArrangement = Arrangement.spacedBy(7.dp)
+        ) {
+            items(badges) { badge ->
+                if (badge.hasObtained) {
+                    MyBadgesObtainedCard(info = badge, onClick = { onClickCard(badge) })
+                } else {
+                    MyBadgesNotObtainedCard(onClick = { onClickCard(badge) })
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun MyBadgesObtainedCard(
+    info: MyBadges,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Card(
+        modifier = modifier
+            .aspectRatio(1f)
+            .clickable { onClick() },
```

Comment:
```suggestion
    Card(
        modifier = modifier
            .aspectRatio(1f)
            .clip(RoundedCornerShape(10.dp)
            .clickable { onClick() },
```

이런식으로 클립을 적용해 리플효과를 아이템에 딱 맞게 할 수 있습니다. 잘 안되면 패딩 위치를 옮겨보면 될거에요
현재 아이템 모양이랑 리플효과 모양이랑 달라서 맞춰야 될 것 같습니다~